examples/labs/rover_controller/swap_joysticks.py

```diff
@@ -0,0 +1,18 @@
+from pitop import Pitop, DriveController, Camera, PanTiltController
+from pitop.labs import RoverWebController
+from pitop.labs.web.blueprints.rover import pan_tilt_handler, drive_handler
+
+rover = Pitop()
+rover.add_component(DriveController())
+rover.add_component(PanTiltController())
+rover.add_component(Camera())
+
+rover_controller = RoverWebController(
+    get_frame=rover.camera.get_frame,
+    message_handlers={
```

Comment:
I think this needs the `drive` argument too

Reply:
you don't need to pass it when you override the right_joystick handler, this should be added to the docs though for sure 👍

Reply:
I've updated the docs to say this 👍

Reply:
Oh yeah sorry, missed that!